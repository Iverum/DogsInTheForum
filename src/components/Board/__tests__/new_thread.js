import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import NewThread from '../new_thread'

test('NewThread is rendered correctly', () => {
  const component = renderer.create(
    <NewThread />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
