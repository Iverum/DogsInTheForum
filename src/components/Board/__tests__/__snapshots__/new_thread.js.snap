exports[`test NewThread is rendered correctly 1`] = `<noscript />`;
